codemods/videos.js

```diff
@@ -1,22 +1,36 @@
 const visit = require('unist-util-visit');
 const {
   addAttribute,
-  hasClassName,
   findAttribute,
+  isMdxBlockElement,
 } = require('./utils/mdxast');
 
+const getVideoProps = (src) => {
+  const propsObj = {};
+  const domain = src.split('/')[2];
+  if (domain === 'fast.wistia.net') {
+    propsObj.type = 'wistia';
+    propsObj.id = src.match(/iframe\/([a-zA-Z0-9]+)\??/)[1];
+  }
+  if (domain === 'www.youtube.com') {
+    propsObj.type = 'youtube';
+    propsObj.id = src.match(/embed\/([a-zA-Z0-9]+)\??/)[1];
+  } else {
+    throw new Error('Video type not recognized.');
+  }
+  return propsObj;
```

Comment:
[nit] Rather than creating an object, then filling in its values in the `if` blocks, and returning it, you can just `return` in the blocks themselves. I typically find that a less noisy for these kinds of things.

That might look something like:
```js
const domain = src.split('/')[2];

if (domain === 'fast.wistia.net') {
  return {
    id: src.match(/iframe\/([a-zA-Z0-9]+)\??/)[1],
    type: 'wistia',
  }
}

if (domain === 'www.youtube.com') {
  return {
    id: src.match(/embed\/([a-zA-Z0-9]+)\??/)[1],
    type: 'youtube',
  }
}

throw new Error('Video type not recognized.');
```

Reply:
yep, good call.